File: app/src/main/java/com/lis/player_java/data/room/model/RemoteKeys.java
package com.lis.player_java.data.room.model;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

@Entity
public class RemoteKeys {
    @PrimaryKey
    public Long musicId;
    public int prefKey;
    public int nextKey;

    public RemoteKeys(Long musicId, int prefKey, int nextKey) {
        this.musicId = musicId;
        this.prefKey = prefKey;
        this.nextKey = nextKey;
    }
}

File: app/src/main/java/com/lis/player_java/data/room/model/RemoteKeys.kt
package com.lis.player_java.data.room.model

import androidx.room.PrimaryKey
import com.lis.player_java.data.room.model.AlbumForMusic
import com.lis.player_java.data.room.model.GenreType
import androidx.room.Embedded
import androidx.room.Entity
import com.lis.player_java.data.room.model.MusicDB
import com.lis.player_java.data.room.model.ArtistDB
import com.lis.player_java.data.room.model.ArtistType

@Entity
data class RemoteKeys(
    @PrimaryKey val musicId: Long,
    val prefKey: Int,
    val nextKey: Int
)